{"uuid": "106ce000-a76c-49a8-8eb1-0f49bb0b224c", "children": ["4ed2e250-c562-439c-a7db-2b129ec5d9f1"], "befores": [{"name": "_xunit_setup_method_fixture_TestDossierUpload", "status": "broken", "statusDetails": {"message": "AttributeError: 'super' object has no attribute 'setup_method'\n", "trace": "  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\python.py\", line 821, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\python.py\", line 717, in _call_with_optional_argument\n    func()\n  File \"E:\\AutoTest\\FgAI\\test_001\\test_DossierUp.py\", line 26, in setup_method\n    super().setup_method()  # 调用父类的setup_method\n"}, "start": 1744613172980, "stop": 1744613172982}], "start": 1744613172980, "stop": 1744613173105}